spec/driver_spec.cr

```diff
@@ -49,20 +49,20 @@ describe PlaceOS::Driver::DriverManager do
 
     {{PlaceOS::Driver::CONCRETE_DRIVERS.values.first[1]}}.functions.should eq(%({
       "switch_input":{
-        "input":["String"]
+        "input":[{"type":"string","enum":["HDMI","DisplayPort","HDBaseT"],"title":"Helper::TestDriver::Input"}]
```

Comment:
these members will actually be `hdmi`, `display_port`, `hdbase_t`
might also be worth checking for an attached `Enum::ValueConverter` on the serializable annotation for enums

Reply:
it seems pretty flexible on the parsing: any combination of upper and lowercase characters matches as well as the underscored variations

I'll take a look at the `ValueConverter` was not aware of that

Reply:
So ValueConverter only effects JSON serialisable, I'll make sure that's in place
and I'll underscore the other values for consistency so `Hdmi` and `HDMI` on different enums in different drivers are considered the same value

Reply:
done